starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,43 @@
+body {
+    margin: 0px;
+    font-size: 20px;
+}
+
+html {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px
+}
+
+.main-image {
+    width:100%;
+    position:relative;
+}
+
+.ap-text {
+    left: 0;
+    font-size: 100px;
+    position:absolute;
+    top:50%;
+    left:50%; 
+    text-align:center;
```

Comment:
This 'text-align: center' is doing nothig. If you want to learn more about centering in css, here you have an amazing guide: https://css-tricks.com/centering-css-complete-guide/